tests/components/sensor/test_transport_nsw.py

```diff
@@ -0,0 +1,53 @@
+"""The tests for the Transport NSW (AU) sensor platform."""
+import unittest
+from unittest.mock import patch
+
+from homeassistant.setup import setup_component
+
+from tests.common import get_test_home_assistant
+
+VALID_CONFIG = {'sensor': {
+    'platform': 'transport_nsw',
+    'stop_id': '209516',
+    'route':  '199',
+    'api_key': 'YOUR_API_KEY'}
+    }
+
+
+def get_departuresMock(_stop_id, route, api_key):
+    """Mock TransportNSW departures loading."""
+    data = {
+        'stop_id': '209516',
+        'route': '199',
+        'due': 16,
+        'delay': 6,
+        'real_time': 'y'
+        }
+    return data
+
+
+class TestRMVtransportSensor(unittest.TestCase):
+    """Test the TransportNSW sensor."""
+
+    def setUp(self):
+        """Set up things to run when tests begin."""
+        self.hass = get_test_home_assistant()
+        self.config = VALID_CONFIG
```

Comment:
We don't need to store this as an instance attribute. `VALID_CONFIG` is a constant, so we can use it in the whole module.